vocab/security/vocabulary.yml

```diff
@@ -235,6 +235,19 @@ property:
       - label: ed25519-2020
         url: https://w3c-ccg.github.io/lds-ed25519-2020/
 
+  - id: secretKeyMultibase
+    label: Secret key multibase
+    domain: sec:Multikey
+    range: sec:multibase
```

Comment:
The `sec:multibase` class does not exist in the current vocabulary (unless I miss it in the maze of PR-s). Note that, for a class, it should be `sec:Multibase` (ie, capital 'M')

Reply:
https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L245

^ related (and in the vocabulary already).

I agree regarding the capitalization

Reply:
`sec:multibase` isn't a class, it's a datatype, like `xsd:string`.

Reply:
@iherman @dlongley so, do I need to do anything in this PR, with respect to `sec:multibase` data type?

Reply:
> `sec:multibase` isn't a class, it's a datatype, like `xsd:string`. This needs the same consideration as `sec:cryptosuiteString`.

Then the capitalization makes sense as is, here is another reference to data types in the v2 context for comparison:

- https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v2#L62

Reply:
> `sec:multibase` isn't a class, it's a datatype, like `xsd:string`. This needs the same consideration as `sec:cryptosuiteString`.

Ah! O.k., understood; the missing capitalization is fine then. However, just as for cryptosuite, the datatype must be properly defined in the spec and then the vocabulary yml file must be updated.

I would propose that this PR would include the datatype definitions in the spec and when all these types of PRs are merged I will take care of incorporating the datatype definitions into the vocabulary specification itself in a separate PR.